numbers/logSpace.go

```diff
@@ -7,7 +7,23 @@ import (
 
 //these two values mark the overflow/underflow boundaries for math.Exp(x)
 func LogCanConvert(x float64) bool {
-	return x > 7.09782712893383973096e+02 || x < -7.45133219101941108420e+02
+	return x > 709.782712893383973096e+02 || x < -745.133219101941108420e+02
+}
+
+//AverageLog returns the average of a list of logSpace numbers
+func AverageLog(x []float64) float64 {
```

Comment:
Can we have tests for AverageLog and MidpointLog?  I am not sure I saw them when looking quick